app/views/dashboards/_latest_upload.html.erb

```diff
@@ -1,31 +1,35 @@
-<tr id="row_<%=upload.csv_type%>"
-    class="<%= latest_upload_class(upload) %>"
-    title="<%= latest_upload_title(upload) %>">
-  <td><%= upload.csv_type %></td>
-  <td><%= upload.ok? ? 'Succeeded' : 'Failed' %></td>
-  <td><%= upload.csv.blank? ? '-' : upload.csv %></td>
-  <td><%= upload.created_at.blank? ? '-' : upload.created_at.to_formatted_s(:long_ordinal) %></td>
-  <td><%= upload.user.blank? ? '-' : upload.user.email %></td>
-  <td><%= upload.comment.blank? ? '-' : upload.comment %></td>
-  <td class="text-center">
-    <% if CSV_TYPES_HAS_API_TABLE_NAMES.include?(upload.csv_type) %>
-      <%= link_to 'Fetch', dashboard_api_fetch_path(upload.csv_type),
-                  class: "btn dashboard-btn-success btn-xs",
-                  role: "button",
-                  id: "api_fetch_" + upload.csv_type,
-                  disabled: cannot_fetch_api(upload.csv_type) %>
-    <% end %>
+<!--Prod flag for bah-6852. Remove if/else statement when approved.-->
+<% if production? && upload.csv_type == 'Sec103' %>
```

Comment:
Can something more generic be done instead of a calling out a specific csv_type here?
Similar to how `CSV_TYPES_HAS_API_TABLE_NAMES` is setup
```suggestion
<% if production? && CSV_TYPES_NO_PROD_NAMES.include?(upload.csv_type)%>
```

Reply:
Only if people remember to use it.

Reply:
Changes are in. How does that look?

Reply:
👍